
The drm Makefile complains about CONFIG_X86_CMPXCHG missing. However I
do not care and it compiles without it, yuck.

  - Rene Rebe <rene@exactcode.de>

--- xc/extras/drm/linux/Makefile.vanilla	2005-02-20 23:07:13.254984432 +0000
+++ xc/extras/drm/linux/Makefile	2005-02-20 23:07:20.380901128 +0000
@@ -279,12 +279,6 @@
 	($(VERSION).$(PATCHLEVEL).$(SUBLEVEL)))
 endif
 
-ifdef ARCHX86
-ifndef CONFIG_X86_CMPXCHG
-$(error CONFIG_X86_CMPXCHG needs to be enabled in the kernel)
-endif
-endif
-
 # This needs to go before all other include paths.
 CC += -I$(DRMSRCDIR)
 
